docs/src/pages.js

```diff
@@ -160,6 +118,48 @@ const pages = [
           { pathname: '/components/about-the-lab' },
           { pathname: '/components/alert' },
           { pathname: '/components/autocomplete' },
+          {
+            pathname: '/components',
+            subheader: '/components/data-grid',
+            children:
+              process.env.CONTEXT === 'production'
```

Comment:
What's the purpose? It seems like the objects are identical...

Reply:
Identical until they are not :)